examples/service-clients/odsp-client/shared-tree-demo/package.json

```diff
@@ -35,13 +35,9 @@
 	"dependencies": {
 		"@azure/msal-browser": "^2.34.0",
 		"@fluid-experimental/odsp-client": "workspace:~",
-		"@fluidframework/container-definitions": "workspace:~",
-		"@fluidframework/container-runtime-definitions": "workspace:~",
-		"@fluidframework/core-interfaces": "workspace:~",
-		"@fluidframework/fluid-static": "workspace:~",
 		"@fluidframework/odsp-doclib-utils": "workspace:~",
 		"@fluidframework/odsp-driver-definitions": "workspace:~",
-		"@fluidframework/tree": "workspace:~",
+		"fluid-framework": "workspace:~",
```

Comment:
Updated 'shared-tree-demo' to import from 'fluid-framework' for validation that TSC can consume our trimmed exports.